<launch>

  #<arg name="server" default="localhost"/>

  <node pkg="vrpn_client_ros" type="vrpn_client_node" name="vrpn_client_node" output="screen">
    <rosparam subst_value="true">
      #server: $(arg server)
      server: 172.16.0.7
      port: 3883

      update_frequency: 10.0
      frame_id: world

      # Use the VRPN server's time, or the client's ROS time.
      use_server_time: false
      broadcast_tf: true

      # Must either specify refresh frequency > 0.0, or a list of trackers to create
      refresh_tracker_frequency: 1.0
      #trackers:
      #- FirstTracker
      #- SecondTracker
    </rosparam>
  </node>

  <node pkg="vrpn_client_ros" type="vrpn_listener_multi.py" name="vrpn_listener_multi" args=" --node_name vrpn_listener_multi " output="screen">

    <param name="agent_num" type="int" value="4" />

    <param name="tcp_ip" value="172.16.0.15" />

    <rosparam param="tcp_port">[8888, 9999, 11111, 22222]</rosparam>

    <rosparam param="agent_ip">['172.16.0.66', '172.16.0.176', '172.16.0.184', '172.16.0.98']</rosparam>

    <rosparam param="car_target">[[3.91, 1.05], [1.71, 3.95], [3.87, 3.88], [1.68, 1.03]]</rosparam>

    <param name="LASER_RANGE" type="double" value="1.4" />

    <param name="ROBOT_LENGTH" type="double" value="0.25" />

  </node>


</launch>
